### ISSUE TITLE
content/ api 500: AttributeError: 'Content' object has no attribute 'versions' on GET /api/v1/search/content/

### ISSUE THREAD

**Issue by alikins** [ansible/galaxy#529]
version: bc34ce1ff23fec5ab868817ffb15298f9675fc24  (release/3.0 branch)

For a GET to http://localhost:8000/api/v1/search/content/  I get a 500:

django debug page traceback:

```
Environment:


Request Method: GET
Request URL: http://localhost:8000/api/v1/search/content/

Django Version: 1.11.13
Python Version: 2.7.5
Installed Applications:
('django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.sites',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'django.contrib.admin',
 'django.contrib.humanize',
 'allauth',
 'allauth.account',
 'allauth.socialaccount',
 'allauth.socialaccount.providers.github',
 'allauth.socialaccount.providers.google',
 'allauth.socialaccount.providers.twitter',
 'bootstrapform',
 'djcelery',
 'rest_framework',
 'rest_framework.authtoken',
 'galaxy.accounts',
 'galaxy.main',
 'autofixture',
 'debug_toolbar')
Installed Middleware:
('debug_toolbar.middleware.DebugToolbarMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware')



Traceback:

File "/var/lib/galaxy/venv/lib/python2.7/site-packages/django/core/handlers/exception.py" in inner
  41.             response = get_response(request)

File "/var/lib/galaxy/venv/lib/python2.7/site-packages/django/core/handlers/base.py" in _legacy_get_response
  249.             response = self._get_response(request)

File "/var/lib/galaxy/venv/lib/python2.7/site-packages/django/core/handlers/base.py" in _get_response
  187.                 response = self.process_exception_by_middleware(e, request)

File "/var/lib/galaxy/venv/lib/python2.7/site-packages/django/core/handlers/base.py" in _get_response
  185.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/var/lib/galaxy/venv/lib/python2.7/site-packages/django/views/decorators/csrf.py" in wrapped_view
  58.         return view_func(*args, **kwargs)

File "/var/lib/galaxy/venv/lib/python2.7/site-packages/django/views/generic/base.py" in view
  68.             return self.dispatch(request, *args, **kwargs)

File "/var/lib/galaxy/venv/lib/python2.7/site-packages/rest_framework/views.py" in dispatch
  494.             response = self.handle_exception(exc)

File "/var/lib/galaxy/venv/lib/python2.7/site-packages/rest_framework/views.py" in handle_exception
  454.             self.raise_uncaught_exception(exc)

File "/var/lib/galaxy/venv/lib/python2.7/site-packages/rest_framework/views.py" in dispatch
  491.             response = handler(request, *args, **kwargs)

File "/var/lib/galaxy/venv/lib/python2.7/site-packages/rest_framework/generics.py" in get
  201.         return self.list(request, *args, **kwargs)

File "/galaxy/galaxy/api/views/search.py" in list
  113.         return self.make_response(queryset)

File "/galaxy/galaxy/api/views/base_views.py" in make_response
  241.             return self.get_paginated_response(serializer.data)

File "/var/lib/galaxy/venv/lib/python2.7/site-packages/rest_framework/serializers.py" in data
  742.         ret = super(ListSerializer, self).data

File "/var/lib/galaxy/venv/lib/python2.7/site-packages/rest_framework/serializers.py" in data
  262.                 self._data = self.to_representation(self.instance)

File "/var/lib/galaxy/venv/lib/python2.7/site-packages/rest_framework/serializers.py" in to_representation
  660.             self.child.to_representation(item) for item in iterable

File "/galaxy/galaxy/api/serializers/roles.py" in to_representation
  60.             .to_representation(instance))

File "/var/lib/galaxy/venv/lib/python2.7/site-packages/rest_framework/serializers.py" in to_representation
  504.                 ret[field.field_name] = field.to_representation(attribute)

File "/var/lib/galaxy/venv/lib/python2.7/site-packages/rest_framework/fields.py" in to_representation
  1816.         return method(value)

File "/galaxy/galaxy/api/serializers/roles.py" in get_summary_fields
  139.             for g in obj.versions.all()]

Exception Type: AttributeError at /api/v1/search/content/
Exception Value: 'Content' object has no attribute 'versions'
```

**Comment by alikins** [comment]
ping @cutwater @chouseknecht

**Comment by chouseknecht** [comment]
Fixed by #531
